src/tweenable.js

```diff
@@ -270,6 +271,12 @@ export const composeEasingObject = (
   easing = DEFAULT_EASING,
   composedEasing = {}
 ) => {
+  if (Array.isArray(easing)) {
```

Comment:
Since this function is now accepting an Array for `easing`, we should update the JSDoc annotation on line 264:

```js
* @param {Object|string|Function|Array.<number>} [easing]
```

Additionally, we should update the public documentation for `shifty.tweenConfig` similarly:

https://github.com/jeremyckahn/shifty/blob/0e32f18b1ae435d4309991353eee07e7b374b4e8/src/index.js#L92-L96

Reply:
Agreed!